esphome/components/tuya/tuya.cpp

```diff
@@ -243,6 +243,11 @@ void Tuya::handle_command_(uint8_t command, uint8_t version, const uint8_t *buff
       ESP_LOGE(TAG, "LOCAL_TIME_QUERY is not handled");
 #endif
       break;
+    case TuyaCommandType::VACUUM_MAP_UPLOAD:
+      this->send_command_(
+          TuyaCommand{.cmd = TuyaCommandType::VACUUM_MAP_UPLOAD, .payload = std::vector<uint8_t>{0x01}});
+      ESP_LOGW(TAG, "Vacuum map upload requested, responding that it is not enabled.");
```

Comment:
This probably doesnt need to be logged as a warning here
```suggestion
      ESP_LOGV(TAG, "Vacuum map upload requested, responding that it is not enabled.");
```

Reply:
Nevermind, just noticed multiple other commands do similar.